start_ocean.sh

```diff
@@ -233,6 +234,7 @@ while :; do
 	        printf $COLOR_Y'Starting without IPFS...\n\n'$COLOR_RESET
             ;;
         --with-thegraph)
+            export RUST_LOG=${RUST_LOG:-INFO}
```

Comment:
can we move this above, where all other _LOG_LEVELS statements are located ?

Reply:
I declared the log levels for each component and where they were separated with `#`. Should I define the log levels at the top of the script?

Reply:
yes, have a section with all envs related to logging.   maybe around line 43 in start_ocean.sh